userguide/content/en/docs/get-started/basic-configuration.md

```diff
@@ -37,16 +37,30 @@ time_format_blog = "2006.01.02"
 
 ## Search
 
-By default, the Docsy example site uses its own [Google Custom Search Engine](https://cse.google.com/cse/all). To disable this site search, delete or comment out the following lines:
+By default, the Docsy example site uses its own [Google Custom Search Engine](https://cse.google.com/cse/all). To enable or disable this site search, add rsp. delete/comment out the following line that starts with `gcs_engine_id`:
```

Comment:
I would suggest reverting the text in this section. This section's purpose is to explain how to disable GCSE, and the original text served that purpose clearly relative to the example site's config file.

Reply:
I see the point. I restored the section and changed the wording here a bit. Looks good to me now.